hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILECrck.py

```diff
@@ -43,16 +47,19 @@ def __init__(self, **kwargs: Any) -> None:
         # Define dictionary with default params that can be overridden from subclasses or CLI
         default_kwargs = dict(
             # declared in BaseMIL:
-            pooling_type=GatedAttentionLayer.__name__,
+            pooling_type=AttentionLayer.__name__,
+            encoding_chunk_size=60,
+            cache_mode=CacheMode.MEMORY,
+            precache_location=CacheLocation.CPU,
             # declared in DatasetParams:
-            local_datasets=[Path("/tmp/datasets/TCGA-CRCk")],
-            azure_datasets=["TCGA-CRCk"],
+            local_dataset=Path("/tmp/datasets/TCGA-CRCk"),
```

Comment:
These 3 variables will need to be updated or this will not run. The new names are:
- local_datasets (and the value should be a list containing that path)
- azure_datasets (and the value should be a list containing that string)
- max_epochs